wavefront/server/modules/voice_agents_module/voice_agents_module/services/exotel_service.py

```
@@ -0,0 +1,106 @@
+        Returns:
+            dict: Call details including call_sid and status
+        """
+        try:
+            # status_callback_url = f'{self.call_processing_base_url}/webhooks/exotel/status'
+
+            # Construct Exotel v1 API endpoint (append .json for JSON response)
+            endpoint = f'https://{api_key}:{api_token}@{subdomain}/v1/Accounts/{account_sid}/Calls/connect.json'
+
+            app_id = os.getenv('EXOTEL_APP_ID')
+            if not app_id:
+                raise ValueError('EXOTEL_APP_ID environment variable is not set')
```

Ruff (0.14.14)

[warning] Abstract `raise` to an inner function

(TRY301)